spk/owncloud/src/service-setup.sh

```diff
@@ -145,7 +139,7 @@ service_postinst ()
 
     if [ "${SYNOPKG_PKG_STATUS}" = "INSTALL" ]; then
         # Parse data directory
-        DATA_DIR="/volume1/${wizard_data_share}/data"
+        DATA_DIR="${SYNOPKG_PKGDEST_VOL}/${wizard_data_share}/data"
```

Comment:
with the shared folder redesign you can simply use:
```suggestion
        DATA_DIR="${SHARE_PATH}/data"
```

Reply:
Thanks, will try that approach.